src/app/store/reducers/score.reducer.ts

```diff
@@ -18,13 +18,17 @@ export const initialState: ScoreState = {
  * @returns {ScoreState}
  */
 export function reducer(state = initialState, action: fromScore.ScoreActions): ScoreState {
-  // TODO #8 : Implement the Add Score
+  // DONE #8 : Implement the Add Score
 
   switch (action.type) {
     case fromScore.RESET_SCORE:
       return {
         scoreValue: 0
       };
+    case fromScore.INCREMENT_SCORE:
+      return {
+        scoreValue: state.scoreValue + 1
```

Comment:
You should use action.payload, the score increment in 20 points but this value comes from the component.